sources/osg/FrameBufferObject.js

```diff
@@ -43,12 +44,13 @@ FrameBufferObject.flushDeletedGLFrameBuffers = function(gl, availableTime) {
     if (availableTime <= 0.0) return availableTime;
 
     if (!FrameBufferObject._sDeletedGLFrameBufferCache.has(gl)) return availableTime;
+    var deleteList = FrameBufferObject._sDeletedGLFrameBufferCache.get(gl);
+    if (deleteList.length === 0) return availableTime;
 
     var elapsedTime = 0.0;
     var beginTime = Timer.instance().tick();
-    var deleteList = FrameBufferObject._sDeletedGLFrameBufferCache.get(gl);
+    deleteList = MACROUTILS.arrayUniq(deleteList);
```

Comment:
Do you mean for FBO only or all the gl resource delete list ?
Would make sense to me to use Set to prevent multiple shared GL resources delete
Need perf test (pagedlod sample I think)
Can change.
@jtorresfabra any advice ?

Reply:
@cedricpinson @Kuranes  I don't think we even need that. deleting something that have already been deleted has no effect. (for instance: https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/deleteFramebuffer ). So I don't think it is worth to change this.

Reply:
Interesting didn't know, thanks, so I remove the uniq changes then and believe in the power of the spec ?
@cedricpinson ok with that ?

Reply:
I did :), and it worked so far...